client/src/components/EventCard.tsx

```diff
@@ -47,14 +49,39 @@ export const EventCard: React.FC<EventCardProps> = ({ event }) => {
       )}
     </>
   );
+
+  const isRunning: boolean =
+    isFuture(new Date(event.ends_at)) && isPast(new Date(event.start_at));
+  const isEnded: boolean = isPast(new Date(event.start_at));
   const eventStatus = event.canceled
     ? 'Canceled'
-    : new Date(event.start_at) < new Date()
-    ? 'Passed'
+    : isRunning
+    ? 'Running'
+    : isEnded
+    ? 'Ended at'
     : 'Upcoming';
```

Comment:
```suggestion
    : isStartInPast
    ? isEndInFuture
      ? EventStatus.running
      : EventStatus.ended
    : EventStatus.upcoming;
```

Reply:
This section `isStartInPast ? isEndInFuture ? EventStatus.running: EventStatus.ended` was hard to read, I had to spend while to understand its logic.

I think it means 

- if it's in the past and ends in the future, then it's running.
- if it isn't running but in the past, then it's ended.
- else it's upcoming

Reply:
I generally often have problems with reading lots of ternary operators. I tried to put there some parenthesis to make clearer which case ends where, but eslint didn't like them. Indentation gives slight hints about what goes where.
 
To summary:
- Is canceled?
  - Yes (canceled) -> canceled
  - No (not canceled), is start in the past?
    - Yes (start in the past), is end in future?
      - Yes (end in future) -> running
      - No (end not in future) -> ended
    - No (start not in the past) -> upcoming

_Simpler_ or _easier for reading_ could be making this into function, and dropping using ternary operator, because that way the we can skip all the explicit `else` branches, returning as soon as possible and not going into deeper indentation. For example:
```typescript
const getEventStatus = ({canceled, isStartDateInPast, isEndDateInFuture}) => {
  if (canceled) return EventStatus.canceled;
  if (!isStartDateInPast) return EventStatus.upcoming;
  if (isEndDateInFuture) return EventStatus.running;
  return EventStatus.ended;
}
```

Reply:
Thank you 💚, I forgot about functions somehow 😓.

This make things easier, and I think moving it become easier, too. If this approved, I am going to move enum and the function to utils folder, then use it in the dashboard and chapter 👍